ci_scripts/pyproject.toml

```diff
@@ -12,7 +12,9 @@ SOURCE_DIR = '../mbed_targets'
 RELEASE_BRANCH_PATTERN = "^release.*$"
 AWS_BUCKET = 'mbed-target'
 MODULE_TO_DOCUMENT = 'mbed_targets'
-DOCUMENTATION_DEFAULT_OUTPUT_PATH = '../docs/user_docs/local_output'
+DOCUMENTATION_DEFAULT_OUTPUT_PATH = '../local_docs'
+DOCUMENTATION_TEMP_CI_OUTPUT_PATH = "../temp_docs"
```

Comment:
Why was this required? I must've missed it before - wonder if specifying `TEMP` as config is necessary.

Reply:
It's so that I can clear out the `docs` dir before adding the new documentation during a release. The reason I made it a config var is that the docs generation script needs to know where to output the docs and the release script needs to know where they have been put to move them into the newly emptied `docs` dir. This felt like the easiest way at the time.